src/test/scala/com/gu/paymentFailure/EndToEndHandlerTest.scala

```diff
@@ -203,7 +240,28 @@ object EndToEndDataWithBillingDetails extends EndtoEndBaseData {
 object EndToEndData extends EndtoEndBaseData {
 
   override val expectedEmailSend =
-    """{"To":{"Address":"john.duffell@guardian.co.uk","SubscriberKey":"john.duffell@guardian.co.uk","ContactAttributes":{"SubscriberAttributes":{"subscriber_id":"A-S00071536","product":"Supporter","payment_method":"CreditCardReferenceTransaction","card_type":"Visa","card_expiry_date":"12/2019","first_name":"eSAFaBwm4WJZNg5xhIc","last_name":"eSAFaBwm4WJZNg5xhIc","paymentId":"2c92c0f95fc912eb015fcb2a481720e6","price":"$49.00","serviceStartDate":"17 November 2017","serviceEndDate":"16 November 2018"}}}}"""
+    """{
+      |  "DataExtensionName": "first-failed-payment-email",
+      |  "To": {
+      |    "Address": "john.duffell@guardian.co.uk",
+      |    "SubscriberKey": "john.duffell@guardian.co.uk",
+      |    "ContactAttributes": {
+      |      "SubscriberAttributes": {
+      |        "subscriber_id": "A-S00071536",
+      |        "product": "Supporter",
+      |        "payment_method": "CreditCardReferenceTransaction",
+      |        "card_type": "Visa",
+      |        "card_expiry_date": "12/2019",
+      |        "first_name": "eSAFaBwm4WJZNg5xhIc",
+      |        "last_name": "eSAFaBwm4WJZNg5xhIc",
+      |        "paymentId": "2c92c0f95fc912eb015fcb2a481720e6",
+      |        "price": "$49.00",
+      |        "serviceStartDate": "17 November 2017",
+      |        "serviceEndDate": "16 November 2018"
+      |      }
+      |    }
+      |  }
+      |}"""
```

Comment:
the build tests is failing because of this missing
```suggestion
      |}""".stripMargin
```